3-JSS/webpack.config.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const CopyPlugin = require('copy-webpack-plugin')
+module.exports = {
+    mode: 'development',
```

Comment:
what does this do, what are the other options?

Reply:
either dev prod mod. I talked to tom now about it.
I'll change it to prod.